src/main/java/uk/gov/companieshouse/api/strikeoffobjections/file/FileTransferApiClient.java

```diff
@@ -167,15 +175,84 @@ public FileTransferApiClientResponse delete(String requestId, String fileId) {
      * @param httpServletResponse The HttpServletResponse to stream the file to
      * @return FileTransferApiClientResponse containing the http status
      */
-    public FileTransferApiClientResponse download(String fileId, HttpServletResponse httpServletResponse) throws IOException {
-        // TODO OBJ-200 replace this dummy response with the implementation.
-        FileTransferApiClientResponse fileTransferApiClientResponse = new FileTransferApiClientResponse();
-        fileTransferApiClientResponse.setFileId(fileId);
-        fileTransferApiClientResponse.setHttpStatus(HttpStatus.OK);
+    public FileTransferApiClientResponse download(String requestId, String fileId, HttpServletResponse httpServletResponse) {
+        String downloadUri = String.format(DOWNLOAD_URI, fileTransferApiURL, fileId);
+
+        return makeApiCall(
+                requestId,
+                //FileTransferOperation
+                () -> restTemplate.execute(
+                        downloadUri,
+                        HttpMethod.GET,
+                        requestCallback -> {
+                            handleRequestCallback(requestCallback);
+                        },
+                        clientHttpResponse -> {
+                            return getClientHttpResponse(httpServletResponse, clientHttpResponse);
+                        }),
+
+                //FileTransferResponseBuilder - the output from FileTransferOperation is the input into
+                // this FileTransferResponseBuilder
+                clientHttpResponse -> {
+                    return getFileTransferApiClientResponse(clientHttpResponse);
+                }
+        );
+    }
+
+    private boolean isFileClean(String requestId, String fileId) {
```

Comment:
Have you tried testing locally and including this call with both a clean and an infected file? Just interested...

Reply:
I would need an infected file in order to do this.

Reply:
'Safe' infected files that contain a virus signature are available. Maybe speak to test team who might know what to use.